src/app/modules/group/components/member-list/member-list.component.ts

```diff
@@ -124,13 +130,13 @@ export class MemberListComponent implements OnChanges, OnDestroy {
               })),
             })));
         } else {
-          return of({
-            columns: descendantTeamsColumns,
-            rowData: [],
-          });
+          return this.getGroupChildrenService.getGroupChildren(groupId, sort, [ 'Team' ])
+            .pipe(map(children => ({
+              columns: groupsColumns,
```

Comment:
The columns are not correct. You may have guessed it (no need to put the type as it is fixed type) and asked me, it would have been easier than let me find the issue.

I've updated the issue.

Reply:
fixed